service/src/test/java/bio/terra/pipelines/configuration/internal/StairwayDatabaseConfigurationTest.java

```diff
@@ -11,7 +11,9 @@
 @SpringBootTest(
     properties = {
       "spring.main.lazy-initialization=true",
-      "datasource.testWithEmbeddedDatabase=false"
+      "datasource.testWithEmbeddedDatabase=false",
+      "pipelines.sentry.dsn=" // piggyback on this test which already supplies properties to test
```

Comment:
maybe we can chat about this - how did you decide on this test class to add this to?

Reply:
i just chose a test that overwrote some properties in the test assplication.conf.  I didnt feel like it was worth creating a whole new class to test this (i would rather remove it and not test than do that).